src/libs/metadata.c

```diff
@@ -408,34 +428,44 @@ static void config_button_clicked(GtkButton *button, dt_lib_module_t *self)
   gtk_grid_attach(GTK_GRID(grid), label, 0, 0, 1, 1);
   label = gtk_label_new(_("hidden"));
   gtk_grid_attach(GTK_GRID(grid), label, 1, 0, 1, 1);
+  gtk_widget_set_tooltip_text(GTK_WIDGET(label),
+            _("tick if the corresponding metadata is not of interest for you"
+              "\nit will be hidden from metadata editor, image information and import modules"
```

Comment:
it will be hidden from metadata editor, image information and import module

Reply:
the s was for the 3 modules ...

Reply:
I'll do it